response/templates/base.html

```diff
@@ -27,8 +27,8 @@
   <!-- Navigation -->
   <nav class="navbar navbar-expand-lg navbar-dark bg-dark static-top">
     <div class="container">
-      <a class="navbar-brand" href="#">
-        <img  id="logo" src="{% static "images/response.png" %}" />
+      <a class="navbar-brand" href="{{ SITE_URL }}/">
```

Comment:
I it would be more robust to rely on [Django URL resolvers](https://docs.djangoproject.com/en/3.0/ref/urlresolvers/) for this, using [the `url` template tag](https://docs.djangoproject.com/en/3.0/ref/templates/builtins/#std:templatetag-url).

All in all, given how it's defined here:

https://github.com/monzo/response/blob/86a3c81b1e485dd5565d844d770a106d0bbe3a97/response/ui/urls.py#L6

You would write as:

```suggestion
      <a class="navbar-brand" href="{% url 'home' %}/">
```

With this, we shouldn't need the context processor you defined.